docs/source/installation.md

```diff
@@ -1,45 +1,77 @@
 # Installation
 
-xcube can be installed from a released conda package, or directly from a
-copy of the source code repository.
+## Prerequisites
 
-The first two sections below give instructions for installation using conda,
-available as part of the [miniconda
-distribution](https://docs.conda.io/en/latest/miniconda.html). If installation
-using conda proves to be unacceptably slow, mamba can be used instead (see
-[Installation using mamba](#installation-using-mamba)).
+xcube releases are packaged as conda packages in the
+[conda-forge](https://conda-forge.org/) channel. It is recommended to install
+xcube into a conda environment using the
+[mamba package manager](https://github.com/mamba-org/mamba), which will
+also automatically install and manage xcube’s dependencies.
+You can find [installation instructions for mamba itself
+here](https://mamba.readthedocs.io/en/latest/installation.html), if you
+don’t already have it installed.
```

Comment:
Do not exclude conda. People may not want to change their package manager. Others already have a conda env and may want to put xcube on top. They cannot simply switch to mamba.

Reply:
Also mention micromamba (I only use micromamba) as an even slimmer version of mamba.

Reply:
@forman  I've rewritten to mention micromamba and to be less emphatic about conda's slowness. But I still think that it's important to mention that conda *is* slow, to avoid user complaints like #856 ("just keeps on solving forever" -- no, probably only for a few hours, but if a user's not expecting that then of course they won't wait that long).